enclave/src/main/java/org/hyperledger/besu/enclave/EnclaveFactory.java

```diff
@@ -48,6 +48,22 @@ public Enclave createVertxEnclave(final URI enclaveUri) {
     return new Enclave(vertxTransmitter);
   }
 
+  public Enclave createVertxEnclave(
+      final URI enclaveUri,
+      final Path privacyKeyStoreFile,
+      final Path privacyKeyStorePasswordFile,
+      final Path privacyAllowlistFile) {
+
+    final HttpClientOptions clientOptions =
+        createTlsClientOptions(
+            enclaveUri, privacyKeyStoreFile, privacyKeyStorePasswordFile, privacyAllowlistFile);
+
+    final RequestTransmitter vertxTransmitter =
+        new VertxRequestTransmitter(vertx.createHttpClient(clientOptions));
+
+    return new Enclave(vertxTransmitter);
+  }
+
```

Comment:
this is a no change, right?

Reply:
This has not changed, it might have moved in the file?